plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -104,13 +116,46 @@ const getPluginContent = async ({wiki, pluginName, reporter, createNode, createC
                 return createWikiNode('text/markdown', url, body);
             }
         }
-        const data = await requestGET({reporter, url: `https://plugins.jenkins.io/api/plugin/${pluginName}`});
-
-        const $ = cheerio.load(data.wiki.content);
-        $('a[id^="user-content"][href^="#"]').remove();
-        data.wiki.content = $.html();
+        if (ASCIIDOC_BLOB_RE.exec(wiki.url)) {
+            const matches = wiki.url.match(ASCIIDOC_BLOB_RE);
+            const url = `https://raw.githubusercontent.com/${matches[1]}/${matches[2]}/${matches[3]}/${matches[4]}`;
+            const body = await requestGET({reporter, url: url});
+            if (body) {
+                return createWikiNode('text/asciidoctor', url, body);
+            }
+        }
+        if (wiki.url.startsWith('https://github.com/') || wiki.url.startsWith('https://www.github.com/')) {
```

Comment:
www.github.com is never used, http://github.com is (just 4 plugins though, also might be better to normalize in update center and not worry about it here)

Reply:
good call, i'll switch it to `new URL('http://github.com').host === 'github.com'` and seperately make an upstream PR to handle http://

Reply:
fixed thanks